### PR TITLE
Build and run the arm example in CI

### PR DESCRIPTION
**Pull request by facundominguez**

### CODE REVIEW

File: examples/arm/qemu-shell.nix [hunk 1]
```diff
@@ -4,8 +4,9 @@ let
 in
 pkgs.mkShell {
   LANG="C.UTF-8";
-  LD_LIBRARY_PATH = "${crossPkgs.buildPackages.gcc-unwrapped}/aarch64-unknown-linux-gnu/lib64";
+  LD_LIBRARY_PATH = "${crossPkgs.buildPackages.gcc-unwrapped.out.lib}/aarch64-unknown-linux-gnu/lib";
   buildInputs = [
     pkgs.qemu
+    pkgs.strace
```
Comment (aherrmann) [inline examples/arm/qemu-shell.nix:10]: Is this still needed or was it only for debugging?

File: .github/workflows/workflow.yaml [hunk 2]
```diff
@@ -51,6 +54,48 @@ jobs:
...
+          nix_path: nixpkgs=./nixpkgs/default.nix
+          extra_nix_config: |
+            trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+            extra-substituters = https://hydra.iohk.io
+      - name: Configure
+        env:
+          BUILDBUDDY_API_KEY: ${{ secrets.BUILDBUDDY_API_KEY }}
+        run: |
+          cat >.bazelrc.local <<EOF
+          # Enabling --config=ci causes cross-compilation to crawl
```
Comment (aherrmann) [inline .github/workflows/workflow.yaml:79]: For reference, this is tracked in https://github.com/tweag/rules_haskell/issues/1524

File: .github/workflows/workflow.yaml [hunk 3]
```diff
@@ -51,6 +54,48 @@ jobs:
             ./bazel-ci-bin/tests/run-tests
             bazel coverage //tests/...
             '
+
+  test-nixpkgs-cross:
+    name: Build & Test - Cross
```
Comment (aherrmann) [inline .github/workflows/workflow.yaml:59]: Could you add this to the [mergify config](https://github.com/tweag/rules_haskell/blob/master/.mergify.yml#L3). You can use the [mergify dashboard](https://dashboard.mergify.io/github/tweag/queues) to make sure the config is valid.

**Review by aherrmann — APPROVED** [review]
Great work, thank you!

File: examples/arm/qemu-shell.nix (hunk 1 above)
Comment (facundominguez) [reply examples/arm/qemu-shell.nix:10]: Only for debugging.
```suggestion
```

**Comment by facundominguez** [comment]
Have I screwed the mergify config? It is not merging this, and the mergify dashboards appears empty.

**Pull request merged**